src/core/Lang.ml

```diff
@@ -146,6 +146,7 @@ let langs_of_filename filename =
   | FT.PL FT.Rust -> [ Rust ]
   | FT.PL FT.R -> [ R ]
   | FT.PL FT.Scala -> [ Scala ]
+  | FT.PL FT.Swift -> [ Swift ]
```

Comment:
What happened here?

Reply:
Can't print out the ast by `-dump_ast` without it.

Reply:
Weird, I know I used `-dump_ast` for Swift all the time. Maybe something got messed up in a refactoring.

Reply:
Probably `-dump_tree_sitter_cst` 😓 Will check.

Reply:
It was `-dump_il` 😆

Reply:
Got it yeah, I don't think I ever used `-dump_il` for Swift.